extensions/ql-vscode/src/data-extensions-editor/queries/java.ts

```diff
@@ -28,7 +28,7 @@ where
   apiName = api.getApiName() and
   supported = isSupported(api) and
   usage = aUsage(api)
-select usage, apiName, supported.toString(), "supported"
+select usage, apiName, supported.toString(), "supported", api.jarContainer(), "jar"
```

Comment:
I wonder if it would make sense to use the same name as for C# `library` or whether there is a specific reason for using `jar` here?

Reply:
There's no specific reason, I'll change it to `library` to match the C# query and the documentation.